packages/uniforms-patternfly/src/__tests__/NumField.test.tsx

```diff
@@ -88,6 +88,13 @@ test("<NumField> - renders an input with correct step (integer)", () => {
   expect(screen.getByTestId("num-field").getAttribute("step")).toBe("1");
 });
 
+test("<NumField> - renders an InputNumber with correct step (set)", () => {
```

Comment:
Please use the same pattern of other tests
```suggestion
test("<NumField> - renders an input with correct step (set)", () => {
```

Reply:
sure - updated